.github/workflows/trunk-hourly-e2e-tests.yml

```diff
@@ -126,9 +125,13 @@ jobs:
 
       - run: ls -l ./cypress/reports/mochareports/
       - name: Upload results
-        run: ./.trunk-ci-analytics/bin --token=${{ secrets.TRUNK_DEBUGGER_TOKEN }}  --input="cypress/reports/**/*junit*.xml"
+        uses: trunk-io/trunk-analytics-uploader@v0.1.0
+        with:
+          junit_paths: cypress/reports/**/*junit*.xml
+          org_url_slug: trunk-staging-org
```

Comment:
We still want to use our org slug right?

Reply:
Yep.